config/runtime.exs

```diff
@@ -139,18 +139,53 @@ end
 ###################################
 
 if System.get_env("RELEASE") == "central_backend" or config_env() == :dev do
-  {:ok, currency_config_json} =
-    "./apps/game_backend/priv/currencies_rules.json"
-    |> File.read()
-
-  config :game_backend, :currencies_config, Jason.decode!(currency_config_json)
-
   {:ok, daily_rewards_json} =
     Application.app_dir(:game_backend, "priv/daily_rewards_rules.json")
     |> File.read()
 
   config :game_backend, :daily_rewards_config, Jason.decode!(daily_rewards_json)
 
+  arena_prestige_ranks_json =
```

Comment:
Arena? I think it should be curse of mirra (we'll translate to champions some day 😂 )

Reply:
And that's why we should use `arena` in the codebase, it will always be an arena. Regardless of what name it used outside